tests/bsim/bluetooth/audio/src/bap_scan_delegator_test.c

```diff
@@ -914,7 +954,16 @@ static void test_main_server_sync_server_rem(void)
 	/* Set the BIS sync state */
 	sync_all_broadcasts();
 
-	/* Remote all sources, causing PA sync term request to trigger */
+	uint32_t bis_sync_req[CONFIG_BT_BAP_BASS_MAX_SUBGROUPS];
+
+	for (size_t i = 0U; i < ARRAY_SIZE(bis_sync_req); i++) {
+		bis_sync_req[i] = 0;
+	}
```

Comment:
```suggestion
	uint32_t bis_sync_req[CONFIG_BT_BAP_BASS_MAX_SUBGROUPS] = { 0 };
```

Should also work if you want a shorter version